docs/index.html

```diff
@@ -6,6 +6,25 @@
   <meta name="viewport" content="width=device-width, initial-scale=1">
   <title>Cerberus - Patterns for Responsive HTML Email Templates</title>
   <link rel="stylesheet" href="assets/cerberus-docs.css">
+  <!-- favicon -->
+  <link rel="shortcut icon" href="assets/favicon/favicon.ico" type="image/x-icon" />
```

Comment:
@douglasprofessional I think the `ico` might be the only one we want.

Some context:
This repo doesn't get much mobile traffic and I'm not especially concerned with folks bookmarking this on their devices. In the interest of [keeping this simple](http://tedgoas.github.io/Cerberus/#contribute), I vote to start with a single `ico` favicon file to address browsers.

How does that sound? If you agree, would you like to modify this PR? Or would you like me to help?